collectRatesData.js

```diff
@@ -0,0 +1,53 @@
+const axios = require('axios')
+
+const config = require('./config');
+const helpers = require('./helpers');
+
+const logger = helpers.logger
+
+const runLogic = () => {
+    logger.info('✋ Init the collect rates data')
+
+    helpers.writeDataFileFromJson(config.rates)
```

Comment:
It's strange that you write to the file something from config.
You can just write the initial state of rates as a const in this file

Reply:
done